spec/controllers/posts_controller_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'rails_helper'
+
+RSpec.describe PostsController, type: :request do
+  describe 'post controller' do
+    context 'index action' do
+      it 'render index action' do
+        get '/users/24/posts/'
+        expect(response).to render_template(:index)
+      end
+
+      it 'return status code ok' do
+        get '/users/24/posts/'
+        expect(response).to have_http_status(:ok)
+      end
+
+      it 'return index view text' do
+        get '/users/24/posts/'
+        expect(response.body).to include('Posts index method')
+      end
+    end
+
+    context 'show action' do
+      it 'render index action' do
+        get '/users/24/posts/123'
+        expect(response).to render_template(:show)
+      end
+
+      it 'return status code ok' do
+        get '/users/24/posts/123'
+        expect(response).to have_http_status(:ok)
+      end
+
+      it 'return show view text' do
+        get '/users/24/posts/123'
+        expect(response.body).to include('Posts show method')
+      end
+    end
+  end
+end
```

Comment:
- [ ] Running rspec to check the unit tests you wrote throws an error. This will be resolved when rspec gem is installed to your project.

![image](https://user-images.githubusercontent.com/86318284/214616708-d5af6a62-d6c7-405a-94a7-612d6cd3f6a4.png)

- [ ] The unit tests are also supposed to be inside a requests folder inside spc folder. The file structure should be something like this: ```spec/requests/_file_name```

![image](https://user-images.githubusercontent.com/86318284/214617227-9fcebd70-2f45-4953-a235-8ded1e765c34.png)

Reply:
resolved